gradle/spec/dependabot/gradle/file_fetcher_spec.rb

```diff
@@ -60,9 +60,9 @@ def stub_content_request(path, fixture)
       end
 
       it "fetches the main buildfile and subproject buildfile" do
-        expect(file_fetcher_instance.files.count).to eq(2)
+        expect(file_fetcher_instance.files.count).to eq(3)
         expect(file_fetcher_instance.files.map(&:name)).
-          to match_array(%w(build.gradle app/build.gradle))
+          to match_array(%w(build.gradle settings.gradle app/build.gradle))
```

Comment:
Does the fixture actually include these files or should we add a new test with a new fixture?

Reply:
I forgot to add test for this. I'll add test.

Reply:
In my opinion, the fixture of `settings.gradle` is already included because settings.gradle will be parsed to collect list of subprojects.